qiskit_metal/_gui/widgets/bases/QWidget_PlaceholderText.py

```diff
@@ -11,7 +11,7 @@
 # Any modifications or derivative works of this code must retain this
 # copyright notice, and modified files need to carry a notice indicating
 # that they have been altered from the originals.
-"""Ask Zlatko for help on this file"""
+"""Ask Zlatko for help on this file."""
```

Comment:
@zlatko-minev , Do you want this line to still be here?

Reply:
Thanks, @priti-ashvin-shah-ibm . I think we can just change that at this point to """@author: Zlatko Minev""" to indicate the same